src/app/_components/Form/FormCheckbox.tsx

```diff
@@ -1,26 +1,57 @@
 import { Checkbox, type CheckboxProps, Field, Label } from '@headlessui/react'
 import { Check } from '@phosphor-icons/react/dist/ssr'
+import { clsx } from 'clsx'
 
 import { Icon } from '@/components/Icon'
 
 type ExcludedHeadlessUIProps = 'className'
 
 export type FormCheckboxProps = {
   label: string
+  labelSize?: 'sm' | 'md'
 } & Omit<CheckboxProps, ExcludedHeadlessUIProps>
 
-export function FormCheckbox({ label, ...rest }: FormCheckboxProps) {
+const TOUCH_TARGET = {
+  base: 'size-5',
+  augmentingClass: 'p-3.5',
```

Comment:
For me the key name are not easy to understand 😕. `augumentingClass` is the padding to create larger clickable area right? Can we say something like `clickArea`? and `clickAreaOffset`? Something descriptive... Should we also make it consistent with other TOUCH_TARGET (maybe in a separate PR tho)?